core/src/components/toast/toast.scss

```diff
@@ -34,17 +38,19 @@
   --button-color: inherit;
 }
 
-.toast-wrapper {
-  background: var(--background);
```

Comment:
From my point of view .toast-wrapper is the wrong class to apply background and box-shadow (see screenshot).
![chrome_2018-12-11_12-11-42](https://user-images.githubusercontent.com/20501666/49796790-f16bb400-fd3d-11e8-906f-3b442d3df338.jpg)

If it keeps there, this would happen

![chrome_2018-12-11_12-13-03](https://user-images.githubusercontent.com/20501666/49796872-2aa42400-fd3e-11e8-80b5-16ef1df106fa.jpg)

Reply:
same for box-shadow. I HOPE this has no negative impact on something elese

Reply:
okay, my fault... it can also added to the wrapper like before